h2o-parsers/h2o-parquet-compat/h2o-parquet-v17-compat/build.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
 
   testCompile "junit:junit:${junitVersion}"
   testCompile project(path: ":h2o-core", configuration: "testArchives")
+  testCompile project(":h2o-jetty-${defaultJettyVersion}")
```

Comment:
@michal-raska I think this dependency should be `testRuntimeOnly` as we probably don't need to compile tests against it.

... but anyway, do we really need it? It is already packaged into runtime jar by `test-package-java` so this should not be necessary at all

Reply:
Please see https://github.com/h2oai/h2o-3/pull/3115#issuecomment-444778813

- We should not use specific versions outside of fat jars
- I think we decided to remove `defaultJettyVersion` to avoid confusion

Reply:
Can we get around this? It is certainly not convenient that every single module needs to declare a testCompile dependency. I would almost say that is not correct :)